mkdocs.yml

```diff
@@ -80,6 +80,7 @@ pages:
   - 'Schnorr Signatures': 'research/schnorr-signatures.md'
   - 'Miscellaneous Improvements': 'research/miscellaneous-improvements.md'
 - About:
+  - 'Support': 'support.md'
```

Comment:
Is the page 'Support' or 'Support Directory'?  If it is support the page itself really should point to github issues as well.  If it is a support directory, it should be called that in the sidebar/index.